department-of-veterans-affairs/vets-website
# Pensions: Update copy for dependent children

## Summary

- Update copy to remove "your" from Dependent Child name fields
- Update copy to remove "your" from Dependent Child/Who they live with info

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the pension_form_enabled flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run vets-website and vets-api

## How to verify
1. Go to http://localhost:3001/pension/application/527EZ in your browser
2. Select 'Start your application without signing in'
3. Navigate through the form until you reach "Do you have any dependent children?". Select "yes."
4. The "Dependent Child" page should say "Child's first name," etc., and not "Your first name."
5. Fill out the page and continue until you reach "Does your child live with you?". Select "no."
6. On the next page, under "Who do they live with?", you should see "First name," etc., and not "Your first name."

## Related issue(s)

- [Link to Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73291)
- [Link to Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73288)
- [Link to Epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

- Completed the form and noted that the child's name form did not say "Your name" and that the address form did not say "Your address".

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Address |    <img width="344" alt="Address-before" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/b97b9970-d95e-49bd-9577-e727b5de4638">    |    <img width="509" alt="Address-after" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/ef12d445-3caa-4a75-b00d-65db60b2eded">   |
| Name | <img width="524" alt="Child-before" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/a5dac44d-598e-4421-9288-ec6b887915a1"> | <img width="524" alt="Child-after" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/87936962-7566-4155-8772-379111c7ce4f"> |

## What areas of the site does it impact?

Pension Benefits

## Acceptance criteria
- [Link to Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73291)
- [Link to Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73288)

### Quality Assurance & Testing


### Error Handling


### Authentication
